llvm/test/DebugInfo/AArch64/DW_AT_APPLE_enum_kind.ll

```diff
@@ -1,5 +1,6 @@
-; RUN: llc < %s -filetype=obj -o %t
-; RUN: llvm-dwarfdump -v %t | FileCheck %s
+; RUN: clang++ %s -c -g -emit-llvm -o %t.bc
```

Comment:
an LLVM test can't/shouldn't depend on clang. Can you check in the bitcode file instead?

Reply:
Sounds good
Instead of checking in bitcode i roundtrip this file through `llvm-dis`/`llvm-as` instead. Which tests the same codepath